SPECS/mstflint/mstflint.spec

```
@@ -0,0 +1,298 @@
+%global __remake_config 1
+Summary:        Mellanox firmware burning tool
+Name:           mstflint
+Version:        4.21.0
+Release:        2%{?dist}
+License:        GPLv2+ OR BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            https://github.com/Mellanox/%{name}
+Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-1/%{name}-%{version}-1.tar.gz
+Patch4:         add-default-link-flags-for-shared-libraries.patch
+Patch6:         replace-mlxfwreset-with-mstfwreset-in-mstflint-message.patch
+BuildRequires:  boost-devel
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  libcurl-devel
+BuildRequires:  libibmad-devel
+BuildRequires:  libstdc++-devel
+BuildRequires:  libxml2-devel
+BuildRequires:  make
+BuildRequires:  openssl-devel
+BuildRequires:  zlib-devel
+Requires:       python3
+ExcludeArch:    s390 s390x %{arm}
+%if %{__remake_config}
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
+%endif
+
+%description
+This package contains firmware update tool, vpd dump and register dump tools
+for network adapters based on Mellanox Technologies chips.
+
+%prep
+%setup -q
+
+%patch4 -p1
+%patch6 -p1
```

Nit: Could use the equivalent statement listed below

```suggestion
%autosetup -p1
```

---

Done. Thanks!